project2/style.css

```diff
@@ -0,0 +1,64 @@
+.card {
```

Comment:
No font declared on the `body` or `.card`, so this will display in whatever font is the default on the user's system or browser. That is _a lot_ of leeway to provide as a designer, imo.